mantidproject/mantid
# Improve `Stitch1D` documentation

Adds some improvements to `Stitch1D` documentation.

**To test:**

Build the documentation and make sure it describes accurately what the algorithm is doing. Specially, make sure that the note about how errors are propagated is correct.

Fixes #18117.

Release notes have been updated.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
